spec/features/admin/password_reset_spec.rb

```diff
@@ -6,19 +6,37 @@
     ActionMailer::Base.default_url_options[:host] = 'http://example.com'
   end
 
-  scenario 'allows a user to supply an email for the password reset' do
-    user = create(:user, email: 'foobar@example.com', password: 'secret', password_confirmation: 'secret')
-    visit spree.admin_login_path
-    click_link 'Forgot Password?'
-    fill_in 'Email', with: 'foobar@example.com'
-    click_button 'Reset my password'
-    expect(page).to have_text 'You will receive an email with instructions'
+  context 'when the an account with this email address exists' do
+    before { user = create(:user, email: 'foobar@example.com', password: 'secret', password_confirmation: 'secret') }
+
+    scenario 'allow a user to supply an email for the password reset' do
+      visit spree.admin_login_path
+      click_link 'Forgot Password?'
+      fill_in_email
+      click_button 'Reset my password'
+      expect(page).to have_text 'you will receive an email with instructions'
+    end
+
+    scenario 'email field is required' do
```

Comment:
Honestly, I don't see a lot of value in this spec. We are just checking that we wrote `required: true` in the view. Also, is that change required in this PR? I think we should keep it as simple as possible but maybe I'm missing something?

Reply:
I left this test because previously a blank submission would render an error because ActiveRecord is validating that field, and we were testing that functionality in this feature spec.

I made the choice to remove all errors from this page, including that blank field error, but I added `required: true` in the view so that, generally, the behavior submitting the form with a blank field is not possible. 

I also added this because I think it is foolish not to require a field that will result in a failing validation if it is left blank.

We can remove the test, but I thought it was a good idea to leave it there because it validates that a similar behavior still exists.

Thoughts?

Reply:
I'm fine adding the `require: true` but personally, I'd remove this test as is or I'd add an expectation that a client-side validation error is displayed if the email is left blank.